checklist/package.json

```diff
@@ -0,0 +1,74 @@
+{
+  "name": "amp-checklist-bot",
+  "author": "The AMP HTML Authors",
+  "version": "0.0.1",
+  "private": true,
+  "repository": "http://github.com/ampproject/amp-github-apps",
+  "description": "A GitHub app which posts checklists on pull requests.",
+  "engines": {
+    "node": "^12.13.1"
+  },
+  "scripts": {
+    "lint": "eslint **/*.ts",
+    "fix": "eslint **/*.ts --fix",
+    "lint-conflicts": "eslint --print-config app.ts | eslint-config-prettier-check",
+    "build": "tsc",
+    "build:watch": "tsc -w --p tsconfig.json",
+    "start": "probot run ./dist/app.js",
+    "dev": "nodemon",
+    "deploy-tag": "git tag 'deploy-checklist-'`date --utc '+%Y%m%d%H%M%S'`",
```

Comment:
This will be relevant to when you go through https://github.com/ampproject/amp-github-apps/blob/master/DEPLOYMENT.md